stl/inc/xmemory

```diff
@@ -1626,8 +1628,9 @@ _NoThrowFwdIt uninitialized_copy(const _InIt _First, const _InIt _Last, _NoThrow
     auto _UFirst = _Get_unwrapped(_First);
     const auto _ULast = _Get_unwrapped(_Last);
     auto _UDest = _Get_unwrapped_n(_Dest, _Idl_distance<_InIt>(_UFirst, _ULast));
-    _Seek_wrapped(_Dest, _Uninitialized_copy_unchecked(_UFirst, _ULast, _UDest,
-                             bool_constant<_Ptr_copy_cat<decltype(_UFirst), decltype(_UDest)>::_Really_trivial>{}));
+    _Seek_wrapped(_Dest,
+        _Uninitialized_copy_unchecked(_UFirst, _ULast, _UDest,
+            bool_constant<_Memcopy_is_safe<decltype(_UFirst), decltype(_UDest), _Memcopy_cat::_Copy_uninitialized>>{}));

```

Comment:
Same suggestion for `using` here, to make the `_Seek_wrapped` call significantly easier to read.